BatchManagement.json

```diff
@@ -0,0 +1,1618 @@
+{
+  "swagger": "2.0",
```

Comment:
This file is placed at the wrong location. Please move it inside the Batch specifications.